dev-python/nuitka/nuitka-0.6.0.2.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_5 python3_6 )
+inherit distutils-r1
+
+DESCRIPTION="Python to native compiler"
+HOMEPAGE="http://www.nuitka.net"
+SRC_URI="http://nuitka.net/releases/${PN^}-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
```

Comment:
Are you sure those packages are needed at build-time only?

Reply:
Actually, my mistake. This dependencies are RDEPENDS. According to the
[debian pacakge too](http://nuitka.net/deb/stable/xenial/dists/xenial/main/binary-amd64/Packages).